spec.md

```diff
@@ -396,6 +396,12 @@ message CreateVolumeRequest {
   // This field is OPTIONAL. The Plugin is responsible for parsing and
   // validating these parameters. COs will treat these as opaque.
   map<string, string> parameters = 5;
+
+  // OPTIONAL field for storing principal information to be used in
+  // the creation of this volume. If the principal is present the
+  // plugin MAY ensure the volume created will be accessible for the
+  // herein specified entities only.
+  Principal volume_root_owner = 6;
```

Comment:
The field may not be different, `volume_root_owner` and `container_acts_as` have to be the same. 
Having said that, i'll remove this part as it does cover a point slightly beyond the core aspect to be fixed here which is to ensure authorization information for shared filesystem backends is provided. I'll add this in a different ticket/PR.